caddyhttp/httpserver/plugin.go

```diff
@@ -447,6 +447,8 @@ var directives = []string{
 	"realip", // github.com/captncraig/caddy-realip
 	"git",    // github.com/abiosoft/caddy-git
 
+	"cache",
```

Comment:
I'm not convinced this is the right place for this directive in the middleware chain. This list is the order in which middlewares are executed: both during set up and during actual request-time (if they inject handlers).

First, things first, to keep track of which directives are third-party plugins, please add a comment to this line with the path to your cache package (as you can see on many of the other lines, sorry if it wasn't explicit in the wiki). `// github.com/nicolasazrak/caddy-cache`

Next, I think we should move it lower in the list... at least probably not "on its own" like this. I wouldn't know for sure where it belongs, but typically you'll want handlers to go as late in the middleware chain as is possible while having it still work properly. If your handler terminates the chain by serving the response, then the request will never see the `rewrite` handler, which might break things if people are using it to rewrite URIs. It also comes before the `errors` directive. Most middleware will want to come _after_ that so that errors returned by your handler can be properly handled. But then again, I don't know for sure in the case of a cache middleware... if you cache the error pages too, then maybe it doesn't matter?

Anyway, I hope that makes sense. If you're unsure, try testing with a few of the directives to see if your cache behaves the way you expect. If not, make sure to move it lower in the chain.